feat: update retrieve.mdx (#112)
storacha/docs: src/pages/how-to/retrieve.mdx

## Using an IPFS HTTP gateway

> You can easily fetch any data stored using Storacha using an IPFS HTTP gateway. Because IPFS is a peer-to-peer, decentralized network, you can use any public HTTP gateway to fetch your data. In this guide, we'll use the gateway at w3s.link (which is optimized for data stored with Storacha), but you can see more worldwide gateways on the [IPFS Public Gateway Checker](https://ipfs.github.io/public-gateway-checker/). To see the full possibile passable parameters, check out [Trustless Gateway Specifications](https://specs.ipfs.tech/http-gateways/trustless-gateway/#http-request).

Reviewer: ```suggestion
You can easily fetch any data stored using Storacha using an IPFS HTTP gateway. Because IPFS is a peer-to-peer, decentralized network, you can use any public HTTP gateway to fetch your data. In this guide, we'll use the gateway at w3s.link (which is optimized for data stored with Storacha), but you can see more worldwide gateways on the [IPFS Public Gateway Checker](https://ipfs.github.io/public-gateway-checker/). To see the full set of parameters and options supported by our gateway, please check out the [Trustless Gateway Specification](https://specs.ipfs.tech/http-gateways/trustless-gateway/#http-request).
```

| `<FILE_NAME>` | The _name_ of the file that you originally uploaded to Storacha. | `example.txt` |
| `<OUTPUT_FILE>` | The path and filename that you want curl to save the file to. This can be different to `<FILE_NAME>`. | `./output-file.txt` |

> We support downdloading flat files out of the box, to download an extra large file or a directory, try passing `dag-scope=entity` in your request to not traverse recursively:

Reviewer: ```suggestion
We support downloading flat files out of the box. When downloading an extra large file or a directory, you may need to use `dag-scope=entity` in your request to avoid fetching the entire upload CAR:
```

```
https://bafybeie2bjap32zi2yqh5jmpve5njwulnkualcbiszvwfu36jzjyqskceq.ipfs.w3s.link/?format=car&dag-scope=entity
```

> To see the full possibility of params, check out [Trustless Gateway Specifications](https://specs.ipfs.tech/http-gateways/trustless-gateway/#http-request).

Reviewer: ```suggestion
To see a full list of supported gateway options, check out [Trustless Gateway Specifications](https://specs.ipfs.tech/http-gateways/trustless-gateway/#http-request).
```
